core/src/Revolution/Processors/Security/Access/Permission/GetList.php

```diff
@@ -44,15 +45,22 @@ public function initialize()
      */
     public function prepareQueryBeforeCount(xPDOQuery $c)
     {
+        $n = $this->modx->newQuery($this->classKey);
+        $n->select([
+            'total' => 'COUNT(DISTINCT `modAccessPermission`.`name`)'
+        ]);
+
         $c->leftJoin(modAccessPolicyTemplate::class, 'Template');
-        $c->query['DISTINCT'] = 'DISTINCT';
+
         $query = $this->getProperty('query', '');
         if (!empty($query)) {
-            $c->where([
-                'modAccessPermission.name:LIKE' => '%' . $query . '%',
-            ]);
+            $query = ['modAccessPermission.name:LIKE' => '%' . $query . '%'];
+            $c->where($query);
+            $n->where($query);
```

Comment:
duplicate line

Reply:
Yes, it's the main query vs. the total count ($n) one. If that bothers you, I can change the variable name. Let me know...

Reply:
When it comes to readability of the code, there's a perfect balance somewhere between simple variable names like `$c` and `$accessPermissionNameQueryInstance`. :D 

Because of the similarity I probably would've named it slightly differently (perhaps `$totalQuery` to indicate its purpose) but I don't feel very strongly about it so no need to change imo.

Reply:
I debated about this detail before you mentioned it. I went ahead and applied your suggestion; honestly I normally make my vars descriptive so I'm not sure why I landed on $n ... maybe matching the style of $c.